services/api-server/src/simcore_service_api_server/services/webserver.py

```diff
@@ -372,7 +373,7 @@ async def update_project_metadata(
         )
         response.raise_for_status()
         data = Envelope[ProjectMetadataGet].parse_raw(response.text).data
-        assert data  # nosec
+        assert data is not None  # nosec
```

Comment:
I though at some point that it might make sense to have an Envelope model which does not allow `None`. Normally I don't like creating special cases, but since this case is so common it would be less code to simply have a special model and avoid all the asserts. Also, the model would detect issues in the deployed system whereas the asserts do not. What do you think?

Reply:
as discussed.